drivers/sensor/tdk/icm42688/icm42688_rtio_stream.c

```diff
@@ -292,12 +292,21 @@ void icm42688_fifo_event(const struct device *dev)
 	struct icm42688_dev_data *drv_data = dev->data;
 	struct rtio_iodev *spi_iodev = drv_data->spi_iodev;
 	struct rtio *r = drv_data->r;
+	uint64_t cycles;
+	int rc;
 
 	if (drv_data->streaming_sqe == NULL) {
 		return;
 	}
 
-	drv_data->timestamp = k_ticks_to_ns_floor64(k_uptime_ticks());
+	rc = sensor_clock_get_cycles(&cycles);
+	if (rc != 0) {
+		LOG_ERR("Failed to get sensor clock cycles");
+		rtio_iodev_sqe_err(drv_data->streaming_sqe, err);
```

Comment:
This driver no longer builds due to use of `err` instead of what I assume was meant to be `rc`. Will open PR to fix.

Reply:
```bash
zephyr/drivers/sensor/tdk/icm42688/icm42688_rtio_stream.c:305:61: error: 'err' undeclared (first use in this function)
  305 |                 rtio_iodev_sqe_err(drv_data->streaming_sqe, err);
      |                                                             ^~~           
```